modules/ROOT/pages/dataweave-cookbook-flat-file-read-and-write.adoc

```diff
@@ -0,0 +1,280 @@
+= Read and Write a Flat File
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, xml, json, dataweave, data weave, datamapper, dwl, dfl, dw, flat file
+
+The following examples show you how to read the contents of a flat file, and also how to write a flat file following the specified schema. Each example consists of a Mule application that uses DataWeave to read and write flat files.
+
+When you work with flat files, configure the flat file schema to use for reading and writing flat file content. The flat file schema defines the structure of a flat file, and this enables Mule to write and read in this format.
+
+These example applications are configured to work with the sample flat file data and the example flat file schema provided in xref:dataweave-flat-file-schemas#full_schema_ex[Flat File Schemas].
+
```

Comment:
Maybe add any Prerequisites around understanding how to configure properties that define paths and how to add a connector to an app so that its dependency appears in the POM file and it and its editable fields will appear as they should in the UI.